src/components/Calculator.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+const Calculator = () => (
+  <>
+    <div className="calculatorWrapper">
+      <div className="displayScreen">
+        <div>
+          <span>0</span>
+        </div>
+      </div>
+      <div className="buttonsContainer">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+        <button type="button">&divide;</button>
+      </div>
+      <div className="buttonsContainer">
```

Comment:
- [ ] According to the project requirements and the feedback from the previous reviewer, could you please consider incorporating at least one child component into your calculator.js file? You might find it helpful to utilize either the calculatorbutton as a child component or the buttonscontainer.

   ![image](https://github.com/CobbyElsonfx/Math-magician/assets/107952724/1f549520-0611-416c-9a78-63dd83d468cf)